File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.py
class Solution:
    def productExceptSelf(self, nums: list[int]) -> list[int]:
        N: int = len(nums)
        ans: list[int] = [1] * N
        for i in range(N):
            for j,n in enumerate(nums):
                if j != i:
                    ans[i] *= n
        return ans


        

File: Data Structures & Algorithms/concatenation-of-array/submission-0.go
func getConcatenation(nums []int) []int {
    n := len(nums)
	ans := make([]int, 2*n)
	// for i:=0; i < n; i++ {
	for i, val := range nums {
		ans[i] = val
		ans[n + i] = val
	}
	return ans
}
